### DISCUSSION TITLE
Is there a way to change the color of the executable code cells?

### DISCUSSION THREAD

**Discussion by dprada** [executablebooks/meta#667] [Q&A] [answered]
Dear developers and community of users:

The default behavior when Jupyter Notebooks are compiled with sphinx into HTML documentation is showing the code cells with a light grey background and a green thick line on the left.
¿Is there a way to change the color of the left thick line? ¿and the background?

Thanks in advance.
And thanks again for your work. This is a great tool.

**Comment by choldgraf** [comment, marked as answer]
Your easiest bet would be to define some CSS rules that are applied to your book. 

[Here's a guide for adding custom CSS with the book](https://jupyterbook.org/advanced/html.html?highlight=css#custom-css-or-javascript).

You'll then need to figure out the selector + rule that will make the change you need. Currently, the cell inputs can be matched with a selector like `div.cell div.cell_input`. So to change the color of the left bar, you could add a selector like this:

```
div.cell div.cell_input {
    border-left-color: red;
}
```

Note that this will break if the HTML structure of notebook cells changes in Jupyter Book (which it might, see [this issue for a proposal to change the HTML structure](https://github.com/executablebooks/MyST-NB/issues/389))
